.travis.yml

```diff
@@ -3,7 +3,7 @@ python: 2.7
 env:
   #- TOX_ENV=py34
   - TOX_ENV=py33
-  - TOX_ENV=py32
+  # - TOX_ENV=py32
```

Comment:
Yep, 3.2 is fine to remove. Could you delete the commented-out lines 3 and 4? I'm not sure why I left in commented-out code.

If I get a chance, when rolling the next release, I'll see if pytest-ordering "just works" with py34 and py35.